src/l500/l500-device.cpp

```diff
@@ -619,6 +619,51 @@ namespace librealsense
         return command{ ivcam2::FRB, 0x0011E000, 0x3f8 };
     }
 
+    bool l500_device::is_special_command( const std::vector< uint8_t > & input )
+    {
+        std::string command_str( input.begin(), input.end() );
+
+        if( command_str == "GET-NEST" )
+        {
+            // Handle extended temperature command
+            auto nest_response = _hw_monitor->send( command{ ivcam2::TEMPERATURES_GET } );
+            if( nest_response.size() != sizeof( extended_temperatures ) )
+            {
+                throw invalid_value_exception( "Extended temperatures get FW command failed" );
```

Comment:
Make clearer: add the size expected vs received, etc.

Reply:
Done